diff -uraN libunwind-1.8.1/configure.ac omv-libunwind-1.8.1/configure.ac
--- libunwind-1.8.1/configure.ac	2024-02-21 14:40:20.000000000 +0100
+++ omv-libunwind-1.8.1/configure.ac	2024-08-30 17:51:08.916216809 +0200
@@ -366,6 +366,9 @@
 AM_CONDITIONAL(OS_QNX, expr x$target_os : xnto-qnx >/dev/null)
 AM_CONDITIONAL(OS_SOLARIS, expr x$target_os : xsolaris >/dev/null)
 
+AC_CHECK_HEADERS([execinfo.h], [have_execinfo_h=yes], [have_execinfo_h=no])
+AM_CONDITIONAL(HAVE_EXECINFO_H, expr x$have_execinfo_h : xyes >/dev/null)
+
 AC_MSG_CHECKING([for ELF helper width])
 AS_CASE([${target_arch}],
         [arm|hppa|ppc32|x86|sh],           [use_elf32=yes; AC_MSG_RESULT([32])],
diff -uraN libunwind-1.8.1/tests/Makefile.am omv-libunwind-1.8.1/tests/Makefile.am
--- libunwind-1.8.1/tests/Makefile.am	2024-02-21 14:39:50.000000000 +0100
+++ omv-libunwind-1.8.1/tests/Makefile.am	2024-08-30 17:52:32.553446546 +0200
@@ -138,8 +138,10 @@
 
 if OS_LINUX
 if BUILD_COREDUMP
+if HAVE_EXECINFO_H
  check_SCRIPTS_cdep += run-coredump-unwind
  noinst_PROGRAMS_cdep += crasher test-coredump-unwind
+endif # GLIBC
 
 if HAVE_LZMA
  check_SCRIPTS_cdep += run-coredump-unwind-mdi
