diff -Naur libunwind-1.2/configure.ac libunwind-1.2.tpg/configure.ac
--- libunwind-1.2/configure.ac	2017-01-13 16:00:44.000000000 +0000
+++ libunwind-1.2.tpg/configure.ac	2017-01-17 19:17:36.941434723 +0000
@@ -161,6 +161,9 @@
 AM_CONDITIONAL(OS_FREEBSD, expr x$target_os : xfreebsd >/dev/null)
 AM_CONDITIONAL(OS_QNX, expr x$target_os : xnto-qnx >/dev/null)
 
+AC_CHECK_HEADERS([execinfo.h], [have_execinfo_h=yes], [have_execinfo_h=no])
+AM_CONDITIONAL(HAVE_EXECINFO_H, expr x$have_execinfo_h : xyes >/dev/null)
+
 AC_MSG_CHECKING([for ELF helper width])
 case "${target_arch}" in
 (arm|hppa|ppc32|x86|sh) use_elf32=yes; AC_MSG_RESULT([32]);;
diff -Naur libunwind-1.2/tests/Makefile.am libunwind-1.2.tpg/tests/Makefile.am
--- libunwind-1.2/tests/Makefile.am	2017-01-13 15:58:24.000000000 +0000
+++ libunwind-1.2.tpg/tests/Makefile.am	2017-01-17 19:18:03.265615935 +0000
@@ -66,8 +66,10 @@
 
 if OS_LINUX
 if BUILD_COREDUMP
+if HAVE_EXECINFO_H
  check_SCRIPTS_cdep += run-coredump-unwind
  noinst_PROGRAMS_cdep += crasher test-coredump-unwind
+endif # GLIBC
 
 if HAVE_LZMA
  check_SCRIPTS_cdep += run-coredump-unwind-mdi
