diff -up libunwind-1.5-rc2/configure.ac.1~ libunwind-1.5-rc2/configure.ac
--- libunwind-1.5-rc2/configure.ac.1~	2020-05-20 00:30:20.970024706 +0200
+++ libunwind-1.5-rc2/configure.ac	2020-05-20 00:31:37.885330054 +0200
@@ -186,6 +186,9 @@ AM_CONDITIONAL(OS_FREEBSD, expr x$target
 AM_CONDITIONAL(OS_QNX, expr x$target_os : xnto-qnx >/dev/null)
 AM_CONDITIONAL(OS_SOLARIS, expr x$target_os : xsolaris >/dev/null)
 
+AC_CHECK_HEADERS([execinfo.h], [have_execinfo_h=yes], [have_execinfo_h=no])
+AM_CONDITIONAL(HAVE_EXECINFO_H, expr x$have_execinfo_h : xyes >/dev/null)
+
 AC_MSG_CHECKING([for ELF helper width])
 case "${target_arch}" in
 (arm|hppa|ppc32|x86|sh) use_elf32=yes; AC_MSG_RESULT([32]);;
diff -up libunwind-1.5-rc2/tests/Makefile.am.1~ libunwind-1.5-rc2/tests/Makefile.am
--- libunwind-1.5-rc2/tests/Makefile.am.1~	2020-04-28 17:49:14.000000000 +0200
+++ libunwind-1.5-rc2/tests/Makefile.am	2020-05-20 00:30:20.970024706 +0200
@@ -73,8 +73,10 @@ endif
 
 if OS_LINUX
 if BUILD_COREDUMP
+if HAVE_EXECINFO_H
  check_SCRIPTS_cdep += run-coredump-unwind
  noinst_PROGRAMS_cdep += crasher test-coredump-unwind
+endif # GLIBC
 
 if HAVE_LZMA
  check_SCRIPTS_cdep += run-coredump-unwind-mdi
